scene/gui/line_edit.cpp

```diff
@@ -810,6 +810,8 @@ void LineEdit::_notification(int p_what) {
 
 			RID ci = get_canvas_item();
 
+			RenderingServer::get_singleton()->canvas_item_set_clip(ci, true);
```

Comment:
Note that this may negatively affect focus StyleBox drawing, in a manner similar to TextEdit: https://github.com/godotengine/godot/issues/32996

Reply:
Not only the focus stylebox but also children of the node.